parser/sql/dialect/sqlserver/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/SQLServerStatement.g4

```diff
@@ -19,6 +19,7 @@ grammar SQLServerStatement;
 
 import Comments, TCLStatement, StoreProcedure, DALStatement;
 
+
```

Comment:
Why add new line here?

Reply:
Just for debug anltr4 syntax . I will remove it.